### PR TITLE
Add pause/unpause button

### PR DESCRIPTION
**Pull request by pcodes**
We have a pause button! Making this PR knowing that #23 exists.

### CODE REVIEW

File: src/client/Components/Pages/Dashboard/ActionButton.jsx [hunk 1]
```diff
@@ -25,11 +25,33 @@ export default class ActionButton extends React.Component<ActionButtonProps, Act
     (this: any).changeButtonState = this.changeButtonState.bind(this);
   }
 
-  changeButtonState() {
+  async changeButtonState() {
     if (this.state.status === 'active') {
-      this.setState({ status: 'paused', isActive: false });
+      await new Promise((resolve) => this.setState({ status: 'paused', isActive: false }, resolve));
```
Comment (Others) [inline src/client/Components/Pages/Dashboard/ActionButton.jsx:30]: nit: there may be cleaner way to do this

File: src/client/Components/Pages/Dashboard/ActionButton.jsx [hunk 2]
```diff
@@ -25,11 +25,33 @@ export default class ActionButton extends React.Component<ActionButtonProps, Act
...
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        'Access-Control-Allow-Credentials': 'true'
+      },
+      body: JSON.stringify(body)
+    });
+
+    if (!response.ok) {
+      console.log('Failed to make secure POST call');
+      throw new Error('Error making POST call!');
```
Comment (Others) [inline src/client/Components/Pages/Dashboard/ActionButton.jsx:54]: nit: error handling would be nice

File: src/client/Components/Pages/Dashboard/ActionButton.jsx [hunk 3]
```diff
@@ -25,11 +25,33 @@ export default class ActionButton extends React.Component<ActionButtonProps, Act
...
+    const response = await fetch(apiUrl, {
+      method: 'POST',
+      credentials: 'include',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        'Access-Control-Allow-Credentials': 'true'
+      },
+      body: JSON.stringify(body)
+    });
```
Comment (Others) [inline src/client/Components/Pages/Dashboard/ActionButton.jsx:50]: nit: may want to factor this out in future

File: src/server/db/database.router.js [hunk 4]
```diff
@@ -77,6 +79,40 @@ class DatabaseRouter {
       });
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  async sendDeploymentIntentionRequest(req: express.$Request, res: express.$Response) {
```
Comment (Others) [inline src/server/db/database.router.js:83]: This is a stupid place to put this

**Review by Others — APPROVED** [review]
Approved with the understanding you'll move the `sendDeploymentIntentionRequest` somewhere else. Also some nits that you can fix for next PR

File: src/server/db/database.router.js (hunk 4 above)
Comment (pcodes) [reply src/server/db/database.router.js:83]: you're a stupid place to put this

File: src/client/Components/Pages/Dashboard/ActionButton.jsx (hunk 1 above)
Comment (pcodes) [reply src/client/Components/Pages/Dashboard/ActionButton.jsx:30]: I've removed state from the action button, no longer relevant

**Pull request merged**